source/Meadow-Filter-Parse.js

```diff
@@ -160,6 +160,47 @@ const addFilterStanzaToQuery = (pFilterStanza, pQuery) =>
 			}
 			break;
 
+    case 'FBJV':// Filter by JSON Value (left-side AND)
+      {
+      const fields = pFilterStanza.Field.split(',', 2);
+      const jsonField =fields[0];
+      const subFields = fields[1];
+      if (jsonField !== undefined && jsonField.trim() !== "" && subFields !== undefined && subFields.trim() !== "")
```

Comment:
(nit) Just doing a truthy check seems simpler here, since `''` is falsy. See: https://developer.mozilla.org/en-US/docs/Glossary/Falsy